### PR TITLE
Unsub from all FCC emails regardless of unsub link

### LINKED ISSUE #8115: one-click unsubscribing from any FCC email should unsubscribe from all FCC emails
Currently we have two different paths:
freecodecamp.com/unsubscribe-quincy
freecodecamp.com/unsubscribe

I think the most expected response from clicking either of these links would be to unsubscribe from ALL emails - not just one type of email.

### PR DESCRIPTION
**Pull request by butterthebuddha**
Closes #8115 
#### Description

Unsubscribes the user from all three types of emails regardless of the link they use to unsubscribe. Couple of concerns with the original issue that I think need to be addressed:
- This unsubscribes the user from _all 3_ (Quincy's newsletter, notifications, and announcements) types of emails because the original issue specifically mentioned all. Just wanted to ensure that this is the desired behaviour.
- I haven't done this but perhaps the copy should be updated to clearly state that the user was unsubscribed from all the emails (in the case they didn't actually intend to).

### CODE REVIEW

File: server/boot/randomAPIs.js [hunk 1]
```diff
@@ -177,7 +177,11 @@ module.exports = function(app) {
         });
         return res.redirect('/map');
       }
-      return user.updateAttribute('sendMonthlyEmail', false, (err) => {
+      return user.updateAttributes({
+        'sendMonthlyEmail': false,
+        'sendQuincyEmail': false,
+        'sendNotificationEmail': false
```
Comment (BerkeleyTrue) [inline server/boot/randomAPIs.js]: Please remove the quotes around the object properties.

**Comment by raisedadead** [comment]
@QuincyLarson @awesomeaniruddh Since this will unsubscribe from all emails, why do we need three buttons on the settings page?

**Comment by BerkeleyTrue** [comment]
@raisedadead The url unsubscribe's from all email but we still want the user to have fine grain control over which emails they receive.

**Comment by awesomeaniruddh** [comment]
@raisedadead those are for unsubscribing/resubscribing from individual emails.

**Comment by raisedadead** [comment]
ahh cool got it! 😉

**Comment by awesomeaniruddh** [comment]
@BerkeleyTrue done. Shouldn't ESLint have caught that though?

**Comment by BerkeleyTrue** [comment]
@awesomeaniruddh Yes, I've just found the rule http://eslint.org/docs/rules/quote-props. Would you like to put together a PR?

**Comment by awesomeaniruddh** [comment]
Sure, why not.

**Comment by allonhadaya** [comment]
Hi, @awesomeaniruddh... I just happened to be reading over this part of the code and noticed that there are three different unsubscribe routes. Only two of them were updated to this new behaviour. Check out [unsubscribeNotifications](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/master/server/boot/randomAPIs.js#L204). It still only updates its own attribute. Is this intended functionality, or is it also supposed to be changed?

Update: I just read the initial issue that said there were only two unsubscribe routes. There are three... Also, now that the three functions will be identical, maybe it's more maintainable to have one function definition that's referenced three times for the different routes.

Hope that helps!

**Pull request merged**